ega-data-api-dataedge/src/main/java/eu/elixir/ega/ebi/dataedge/service/internal/LocalEGARemoteFileServiceImpl.java

```diff
@@ -67,9 +61,14 @@
     @Autowired
     private DownloaderLogService downloaderLogService;
 
+    @Autowired
+    AuthenticationService authenticationService;
```

Comment:
This variable is not used anywhere in the class.

Reply:
I've removed the variable and the import.